cloud/libvirt/actuators/machine/actuator.go

```diff
@@ -115,7 +115,7 @@ func createVolumeAndDomain(machine *clusterv1.Machine, offset int) error {
 	}
 
 	// Create domain
-	if err = libvirtutils.CreateDomain(name, ignKey, name, name, networkInterfaceName, networkInterfaceAddress, autostart, memory, vcpu, offset, client); err != nil {
+	if err = libvirtutils.CreateDomain(name, ignKey, pool, name, name, networkInterfaceName, networkInterfaceAddress, autostart, memory, vcpu, offset, client); err != nil {
```

Comment:
Although you've added `pool` here, why do we pass `name`, twice?

Reply:
> Although you've added `pool` here, why do we pass `name`, twice?

We actually pass it three times ;).  [`CreateDomain`][1] allows the domain, volume, and host names to be configured seperately, but we opt to use the same value for all three here.

[1]: https://github.com/openshift/cluster-api-provider-libvirt/blob/00b16f152ee24782eb6cdf36d8ca988258d46e49/cloud/libvirt/actuators/machine/utils/domain.go#L440